cli/lockfile.rs

```diff
@@ -154,6 +211,88 @@ impl Lockfile {
       }
     };
   }
+
+  /// Checks the given module is included.
+  /// Returns Ok(true) if check passed.
+  fn check_npm_package(&mut self, package: &NpmResolutionPackage) -> bool {
+    match &self.content {
+      LockfileContent::V1(_c) => {
+        panic!("Locking npm specifiers is not supported in lockfile v1");
```

Comment:
We should upgrade v1 lockfiles in memory to v2 on load, then we only have to deal with one lockfile kind in the code and if the user does `--write` it will automatically upgrade them to v2.